src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -85,13 +94,15 @@ public static DiscoveryProvider defaultCreate(
           .setSnippetFileName(id + "/" + DEFAULT_SNIPPET_FILE)
           .build();
     } else if (id.equals(NODEJS)) {
-      return CommonDiscoveryProvider.newBuilder()
-          .setContext(new NodeJSDiscoveryContext(service, apiaryConfig))
-          .setSnippetSetRunner(
-              new NodeJSSnippetSetRunner<Method>(SnippetSetRunner.SNIPPET_RESOURCE_ROOT))
-          .setSnippetFileName(id + "/" + DEFAULT_SNIPPET_FILE)
+      return ViewModelProvider.newBuilder()
```

Comment:
As per our previous discussion, let's do the switchover after we make the substance changes.